src/content/docs/service-level-management/create-slm.mdx

```diff
@@ -24,10 +24,61 @@ To create and manage service levels requires the [capability for modifying and d
 
 For New Relic organizations that have [multiple accounts](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure): Service levels can only be associated with a single account. If you're trying to create a service level for a workload with entities across multiple accounts, you may want to restructure the workloads so that all of their associated entities are in the same account.
 
+New Relic ingests data in many different ways and from very different sources. Each has its own individual flavor, creating many possibilities on how data is consumed. There are some scenarios where it is impossible to configure Service Levels due to the characteristics of the data:
+
+- **Distributed metrics**. Because data is aggregated, a new NRQL function, called Cumulative Distribution Function, needs to be implemented in NRDB. We are working to enable these types of queries.
+- **Timeslice metrics**. Any query involving `newrelic.timeslice.value` or `apm.*` attributes is unsupported. 
+- **Subqueries**. Subqueries are not supported.
+- **Wildcards in the SELECT clause**. Wildcards are not supported in the attributes provided in the `SELECT` clause. Note that the wildcard expression is supported in the `WHERE` clause.
+- **Addition of sum functions**. While it's possible to use `SELECT sum(attributeA)` or `SELECT sum(attributeA + attributeB)`, the expression `SELECT sum(attributeA) + sum(attributeB)` is not supported.
+
 ## Key concepts for creating SLIs and SLOs [#key-concepts]
 
 Keep in mind these concepts when defining SLIs and SLOs.
 
+### Define your key user experiences
+
+Start thinking of the key user experiences you or your team own at the highest level possible and zoom in until more granularity doesn't provide value. When choosing which SLs to start with, we recommend using a top-down approach, meaning start with the least granular ones, and create more granular ones only if necessary.
+
+First of all, identify a "system boundary", what part of your system your user percevies as a "black box" of functionality:
+- in the case of an API, it could be just a service
+- for a data pipeline, it could be a chain of services necessary to process data end-to-end
+
+Once you have established the "top level SLs", you might find that not all the endpoints of your service behave in the same way, and might want to split it further. For example:
+- login transactions might need higher SLO on errors than browsing one
+- duration of some operation is much higher than the rest 
+
+For example, zooming out, a key user experience at New Relic could be: *a customer sends us telemetry data and that data is later available to be queried in our product API or UI.*
```

Comment:
I'd take out the 'zooming out' and reword; kind of thew me, might be a little too colloquial